jobs/resource_job.go

```diff
@@ -608,34 +624,16 @@ func (JobSettingsResource) CustomizeSchema(s *common.CustomizableSchema) *common
 	s.SchemaPath("task", "sql_task", "alert", "subscriptions").SetRequired()
 	s.SchemaPath("task", "for_each_task", "task", "sql_task", "alert", "subscriptions").SetRequired()
 
-	s.SchemaPath("health", "rules", "metric").SetOptional()
```

Comment:
would this break existing user tf code since these fields are not optional anymore?

Reply:
I think it was not optional on the API level anyway, as it is consistent with the openapi spec. So I don't think it will be a breaking change